From: Julien Rioux <julien.rioux@gmail.com>
Date: Mon, 28 Oct 2013 20:42:43 +0000 (+0100)
Subject: Build: Fix compilation with GNU make 4.0
X-Git-Tag: release/2.17.95-1~4^2
X-Git-Url: http://git.savannah.gnu.org/gitweb/?p=lilypond.git;a=commitdiff_plain;h=1ca9814191d16fd3c571d93035247db039254fc1;hp=025f0236ebab947e6fe2047f20b7f74f4cec3fc4

Build: Fix compilation with GNU make 4.0

Fix "recipes commence before first target" error.

Patch from Thomas Klausner.
---

diff --git a/stepmake/stepmake/po-targets.make b/stepmake/stepmake/po-targets.make
index 8919dab..8a0dd76 100644
--- a/stepmake/stepmake/po-targets.make
+++ b/stepmake/stepmake/po-targets.make
@@ -37,10 +37,10 @@ ifneq ($(strip $(ALL_PO_SOURCES)),)
 	 --keyword=_ --keyword=_f --keyword=_i \
 	 $(XGETTEXT_FLAGS) $(ALL_PO_SOURCES)
 endif
-endif
 	sed -i '1,2d' $(po-outdir)/$(package).po
 	sed -i -e 's/^\# This file is distributed.*/$(sed-header)/' $(po-outdir)/$(package).po
 	sed -i -e 's/^\"Content-Type: text\/plain.*/$(sed-content)/' $(po-outdir)/$(package).po
+endif
 
 
 po-update: po
